parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx

```diff
@@ -184,125 +49,352 @@ function SubtaskCounter({ subtaskIndex, numSubtasks }) {
   }
 }
 
-function SubtaskSubmitButton({ subtaskIndex, numSubtasks, onSubtaskSubmit }) {
+function SubtaskSubmitButton({
+  subtaskIndex,
+  numSubtasks,
+  onSubtaskSubmit,
+  disabled,
+}) {
   return (
     <div style={{ textAlign: 'center' }}>
       <button id="submit-button"
-        style={{ cursor: 'not-allowed', backgroundColor: '#c8c8c8' }}
+        style={
+          disabled ? {cursor: 'not-allowed', backgroundColor: '#c8c8c8'} : {}
+        }
         className="button is-link"
         onClick={onSubtaskSubmit}
+        disabled={disabled}
       >
         {subtaskIndex === (numSubtasks - 1) ? "Submit" : "Next"}
       </button>
     </div>
-  )
+  );
 }
 
-function ChatMessage({ text, agentIdx, annotationQuestion, annotationBuckets, turnIdx, doAnnotateMessage, askReason, responseField, onUserInputUpdate }) {
-  var extraElements = '';
-  var responseInputElement = '';
-  if (doAnnotateMessage) {
-    if (annotationBuckets !== null) { 
-      extraElements = (<span key={'extra_' + turnIdx}><br /><br />
-        <span style={{ fontStyle: 'italic' }} >
-          <span dangerouslySetInnerHTML={{ __html: annotationQuestion }}></span>
-          <br />
-          <Checkboxes turnIdx={turnIdx} annotationBuckets={annotationBuckets} askReason={askReason} onUserInputUpdate={onUserInputUpdate} />
-        </span>
-      </span>)
-    }
-    if (responseField !== null) {
```

Comment:
`responseField` is true/false so we should also condition on if it is true. (I think this is a bug from before)